blog/retention analysis.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
you need to push both the "md" file and the "img" folder into a new folder similar to the "2021-12-12-welcome."